tdvt/tdvt/tdvt_core.py

```diff
@@ -508,6 +508,10 @@ def get_csv_row_data(tds_name, test_name, test_path, test_result, test_case_inde
     if test_result and test_result.test_config:
         test_type = 'logical' if test_result.test_config.logical else 'expression'
 
+    # Truncate long process outputs to keep size of csv down
+    if len(cmd_output) > 1024:
+        cmd_output = cmd_output[:1022] + ".."
```

Comment:
```suggestion
        cmd_output = cmd_output[:1024] + "<output_truncated>"
```
More explicit and we don't need to get fancy if we overrun our arbitrary limit.  I'm open to something even higher as well if you want, 4096?

Reply:
I think I originally picked '..' because that's how we truncate for other columns, but I agree that `<output_truncated>` is clearer. I'll also bump up the character limit as well, I think we really only should care if it gets ridiculous as in the test that started this, where the csv was so large it made analyzing it almost impossible because every test had gigantic process outputs.